src/ui/shared/BrutalistUI.jsx

```
@@ -0,0 +1,929 @@
+    setTimeout(() => setIsGlitching(false), 150);
+    setIsOn(!isOn);
+  };
+
+  return (
+    <div className="flex items-center justify-between w-full max-w-sm border border-[#39FF14]/30 p-3 bg-black">
+      <span className="text-sm font-bold tracking-widest uppercase">{label}</span>
+      <button
+        onClick={toggle}
+        className={`relative w-16 h-8 border-2 border-[#39FF14] flex items-center p-1 transition-colors duration-75 ${isGlitching ? 'translate-x-[1px] translate-y-[1px]' : ''}`}
+        aria-pressed={isOn}
+      >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)